code/__DEFINES/mobs.dm

```diff
@@ -914,28 +914,30 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 #define HEAL_LIMBS (1<<6)
 /// Heals all organs from failing.
 #define HEAL_ORGANS (1<<7)
-/// A "super" heal organs, this refreshes all organs entirely, deleting old and replacing them with new.
+/// replaces any non-beneficial organs
 #define HEAL_REFRESH_ORGANS (1<<8)
+/// A "super" heal organs, this replaces all organs entirely, deleting old and replacing them with new.
+#define HEAL_REPLACE_ORGANS (1<<9)
```

Comment:
The documentation here could be improved, particularly differentiating what "non-beneficial" means

Similarly I think `REFRESH_ORGANS` is pretty poorly named now if my understanding of it is correct

```dm
/// Removes any organs which would be detrimental to the mob 
#define HEAL_CLEAR_DANGEROUS_ORGANS (1<<8)
/// Iterates over all the organs of the mob, deletes them, and supplies fresh ones
#define HEAL_REPLACE_ORGANS (1<<9)
```